modules/using-cluster-compare-live-cluster.adoc

```diff
@@ -84,3 +84,18 @@ No patched CRs <10>
 <8> The CRs that did not match to a corresponding template in the reference configuration.
 <9> The metadata hash identifies the reference configuration.
 <10> The list of patched CRs.
+
+[NOTE]
+====
+You can also get the output in `junit` format by adding `-o junit` to the command. For example:
+[source,terminal]
+----
+$ oc cluster-compare -r <path_to_reference_config>/metadata.yaml -o junit
+----
+
+The `junit` output includes the following result types:
+
+* Passed results for each fully matched template.
+* Failed results for differences found or missing required custom resources (CRs).
+* Skipped results for differences patched using the user override mechanism.
```

Comment:
I like this.  Now, do we need the same content in the other two places you mention the `-o junit` flag, below? (using-cluster-compare-must-gather.adoc and using-cluster-compare-telco-ref.adoc)